Documentation/Global/Image.rst

```diff
@@ -9,73 +9,147 @@ Image ViewHelper `<f:image>`
 
 ..  typo3:viewhelper:: image
     :source: ../Global.json
-    :display: tags,description,gitHubLink,arguments
+    :display: tags,description,gitHubLink
+    :noindex:
 
-..  _typo3-fluid-image-example:
+..  contents:: Table of contents
+
+..  _typo3-fluid-image-src:
+
+Argument src - displaying images from the extension assets
+==========================================================
+
+Images used for design purposes and not content purposes are commonly shipped
+with an extension. These images cannot be edited by backend users. They can be
+displayed using the `<f:image>` ViewHelper using the
+:ref:`src <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-src>`
+argument and the `EXT:` syntax:
+
+..  code-block:: html
+
+    <f:image src="EXT:my_site_package/Resources/Public/images/typo3_logo.png" alt="alt text" />
+
+The assets **must** be stored in folder :path:`Resources/Public/` or a sub
+folder of an extension.
+
+If you installed the extension via Composer **before** it had a folder of that
+exact name, reinstall it so that the symlinks in folder :path:`public/_assets`
+are correctly created for you.
+
+..  note::
+
+    Avoid using image operations, such as the arguments
+    :ref:`width <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-width>` or
+    :ref:`crop <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-crop>`
+    with images included from an extensions' assets. Each
+    non-FAL image operation creates a "fake" FAL record, which may lead to problems.
```

Comment:
Really? Wow. Never heard of them, but OK.

Reply:
There is a large warning in the Class and there have been some issues opened in the past. I did not follow the discussion 100%